packages/flutter_tools/lib/src/commands/build_winuwp.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+
+import 'package:meta/meta.dart';
+
+import '../base/common.dart';
+import '../build_info.dart';
+import '../cache.dart';
+import '../features.dart';
+import '../globals.dart' as globals;
+import '../project.dart';
+import '../runner/flutter_command.dart' show FlutterCommandResult;
+import '../windows/build_windows.dart';
+import '../windows/visual_studio.dart';
+import 'build.dart';
+
+/// A command to build a Windows UWP desktop target.
+class BuildWindowsUwpCommand extends BuildSubCommand {
+  BuildWindowsUwpCommand({ bool verboseHelp = false }) {
+    addCommonDesktopBuildOptions(verboseHelp: verboseHelp);
+  }
+
+  @override
+  final String name = 'winuwp';
+
+  @override
+  bool get hidden => !featureFlags.isWindowsUwpEnabled || !globals.platform.isWindows;
+
+  @override
+  Future<Set<DevelopmentArtifact>> get requiredArtifacts async => <DevelopmentArtifact>{
+    // TODO(flutter): add a windows_uwp artifact here once that is updated.
```

Comment:
Is there an issue that describes what needs to be done here in some more detail?

Reply:
Done